src/lib/components/table/terra-mat-paginator-intl.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { MatPaginatorIntl } from '@angular/material';
+import { TranslationService } from 'angular-l10n';
+
+@Injectable()
+export class TerraMatPaginatorIntl extends MatPaginatorIntl
+{
+    constructor(private translation:TranslationService)
```

Comment:
```suggestion
    public nextPageLabel:string = '';
    public previousPageLabel:string = '';
    public firstPageLabel:string = '';
    public lastPageLabel:string = '';
        
    constructor(private translation:TranslationService)
```

Damit der leere String nicht immer wieder zugewiesen werden muss.